src/internal/kopia/backup_bases.go

```diff
@@ -15,7 +15,9 @@ import (
 // TODO(ashmrtn): Move this into some inject package. Here to avoid import
 // cycles.
 type BackupBases interface {
-	RemoveMergeBaseByManifestID(manifestID manifest.ID)
+	// MakeAssistBase converts the base with the given item data snapshot ID from
+	// a merge base to an assist base.
+	MakeAssistBase(manifestID manifest.ID)
```

Comment:
nit: `ConvertToAssistBase` might look better.

Reply:
good call! I'm always kind of bad at picking good names for things hahah